Created initial example. (#2)
Dangl-IT/Dangl.XRechnung.Examples: README.md

# Dangl.XRechnung.Examples

> > An online version of the converter is freely accessible at [my blog](https://blog.dangl.me/xrechnung-converter/)  TODO: ADD A WORKING LINK

Reviewer: Just use https://web-gaeb.de/, we've got an XRechnung viewer already there. It's just in the beta version yet, but once the UI PRs are done we'll release it, probably even this week.

> > A more detailed feature description is available [on my website](https://www.dangl-it.com/products/xrechnung/).

Reviewer: ```suggestion
> A more detailed feature description is available [on our website](https://www.dangl-it.com/products/xrechnung/).
```

> This project requires **Visual Studio 2017** or newer. Otherwise, the Dotnet CLI is also supported.

This is a _demonstration_ library intended to show the usage of the Dangl.XRechnung project.
To build this solution, you need to copy four packages into the `./packages` folder, relative to the root of this project:
* Dangl.XRechnung

> * Dangl.Data.Shared

Reviewer: That doesn't sound right😀 I think we need ´Dangl.XRechnung`, `Dangl.AVA` and `Dangl.AVA.IO`. `Dangl.Data.Shared` is on the public NuGet feed, so no need to reference it here.

> If you are already a customer with support contract, please see any of the packages documentation on how to set up the official NuGet feed for **DanglIT** packages.

These packages are not included in this repository.
This project is intended to demonstrate the usage of the Dangl.XRechnung library.

> [Please get in touch with me if you are interested in the libraries](https://www.dangl-it.com/contact/?message=I%27m+interested+in+Dangl.XRechnung.+Please+contact+me.).

Reviewer: ```suggestion
[Please get in touch with us if you are interested in the libraries](https://www.dangl-it.com/contact/?message=I%27m+interested+in+Dangl.XRechnung.+Please+contact+me.).
```

# Dangl.XRechnung.Examples

> > An online version of the converter is freely accessible at [my blog](https://web-gaeb.de/)

Reviewer: ```suggestion
> An online version of the converter is freely accessible [at **WebGAEB**](https://web-gaeb.de/)
```

> A more detailed feature description is available [on our website](https://www.dangl-it.com/products/xrechnung/).

> This project requires **Visual Studio 2017** or newer. Otherwise, the Dotnet CLI is also supported.

This is a _demonstration_ library intended to show the usage of the Dangl.XRechnung project.

> To build this solution, you need to copy four packages into the `./packages` folder, relative to the root of this project:

Reviewer: ```suggestion
To build this solution, you need to copy three packages into the `./packages` folder, relative to the root of this project:
```
